App.js

```diff
@@ -22,6 +23,16 @@ import { Drawer as CustomDrawer } from './src/components'
 import { StatusBarColor } from './src/styles/Colors';
 import { generateDummyData } from './src/redux/Thunks'
 
+Notifications.setNotificationHandler({
+  handleNotification: async () => {
+    return {
+      shouldShowAlert: true,
+      shouldPlaySound: true,
+      shouldSetBadge: true,
+    };
+  },
+});
+
```

Comment:
What does this code do? Do notification alerts appear while the app is open if we set shouldShowAlert to false? Maybe this is the desired behavior.
```suggestion
Notifications.setNotificationHandler({
  handleNotification: async () => {
    return {
      shouldShowAlert: false,
      shouldPlaySound: false,
      shouldSetBadge: true,
    };
  },
});

```

Reply:
Also, this code should be a function of the notifications module.

Reply:
If shouldShowAlert: false the notifications alert doesn't appear with the app open. The notifications code was moved in da9cadbaf09044c09bd762d03fb08d007a64ea5a